src/views/Timesheets/components/SavedShiftsList/index.js

```diff
@@ -83,6 +84,21 @@ export default class SavedShiftsList extends Component {
     });
   }
 
+  componentDidMount() {
```

Comment:
This componentDidMount is doing a lot of stuff. Might be worth adding a little descriptive comment to help us out when we pick up working on it.